tiledb/sm/query/readers/result_tile.cc

```diff
@@ -261,6 +275,36 @@ ResultTile::TileTuple* ResultTile::tile_tuple(const std::string& name) {
   return nullptr;
 }
 
+void ResultTile::wait_all_coords() const {
+  for (auto& at : coord_tiles_) {
+    auto& tile_tuple = at.second;
+    if (tile_tuple.has_value()) {
+      tile_tuple.value().fixed_tile().data();
```

Comment:
Is `tile_tuple` in this function a different type from `tile_tuple` in the function below?
"Wait for all the data to arrive from this tile" looks like it wants to be its own function.

Reply:
Oh you always spot it when I am lazy :D Agreed I should have done that in the first place.